src/core/scene/a-scene.js

```diff
@@ -128,22 +135,32 @@ var AScene = module.exports = registerElement('a-scene', {
      */
     enterVR: {
       value: function (event) {
-        this.setStereoRenderer();
-        if (isMobile) {
-          setFullscreen(this.canvas);
-        } else {
-          this.stereoRenderer.setFullScreen(true);
+        var self = this;
+        var vrSource = [{source: this.renderer.domElement}];
+        if (!vrDisplay) { return; }
+        vrDisplay.requestPresent(vrSource).then(enterVRSuccess, enterVRFailure);
+        function enterVRSuccess () {
+          self.addState('vr-mode');
+          self.emit('enter-vr', event);
+        }
+        function enterVRFailure () {
+          error('enter VR mode error. requestPresent failed');
```

Comment:
What do you mean? The error will be shown in console. Do you mean show on screen?